src/Symfony/Component/Translation/Tests/DependencyInjection/TranslationPassTest.php

```diff
@@ -25,26 +25,33 @@ public function testValidCollector()
         $loader = (new Definition())
```

Comment:
@Nyholm please see https://github.com/symfony/symfony/pull/23667#discussion_r130970072. 
Once BC re-added, you have to keep the legacy test (which uses `translation.loader` service) in addition to the new one (`translation.reader`).